Clarify a sentence on methods (#988)
hadley/r-pkgs: testing-basics.Rmd

If you're familiar with frameworks for unit testing in other languages, you should note that there are some fundamental differences with testthat.
This is because R is, at heart, more a functional programming language than an object-oriented programming language.

> For instance, because R's main object-oriented systems (S3 and S4) are based on generic functions (i.e., a method is the implementation of a generic function for an object of a particular class), testing approaches built around objects and methods don't make much sense.

Reviewer: If we're going to touch this, I think it needs to be tweaked.
```suggestion
For instance, because R's main object-oriented systems (S3 and S4) are based on generic functions (i.e., a method implements a generic function for a specific class), testing approaches built around objects and methods don't make much sense.
```
